### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by enricllagostera** [chrisgrieser/obsidian-quadro#3] [General]
Hi @chrisgrieser ! I just found out about Quadro. The plugin looks super interesting, and I just wanted to say thanks for developing it. I'm also working on some QDA-related FOSS tools within a research group, and I think it is definitely an area for huge potential in qualitative research. I hope to be able to contribute to Quadro a bit in the future, as soon as I have had more time to test it and use it in a couple projects. Cheers!

**Comment by chrisgrieser** [comment]
Thank you for the kind words!

Yeah, I welcome contributions and ideas. Feel free to post about your qda-related tools, if it's a fit they could be integrated with Quadro?

**Reply by enricllagostera** [reply]
I think one thing that would be a good fit with Quadro is the QDPX file format exporting I've been working on. In the future (when my dissertation is done, hehe), I could separate it and integrate it into Quadro. With that, Quadro projects could be made more compatible with existing QDA tools, which would make it easier for folks to migrate projects from NVivo or Atlas.ti (or any other QDPX-exporting tool) to Quadro, and vice-versa.
